src/qml/MessageBubble.qml

```diff
@@ -93,7 +93,7 @@ BorderImage {
         } else if (sending) {
             return "grey"
         } else if (messageIncoming) {
-            return mainView.enableNightMode ? "black" : "white"
+            return ""+theme.palette.normal.background == "#ffffff" ? "white" : "black"
```

Comment:
I'm not very happy with this solution. It works but is there a better way to find out which theme is currently used? Or is there a color in the palette, which we could use as a background.

Currently the background in Suru Dark is **not** "black". It's something around "#111111" and using this would make the message bubble invisible. We need to set it to "#000000". Is this color in the Suru palette? I wasn't able to find a documentation for the theme.palette object...

Reply:
I think you can use `Suru.theme`. I'm not sure if there is something similar for the UITK. Currently the both follow the dark theme so that could be a better solution.

Reply:
I have found the theme.name property. I would say that's a clean way to check the theme :)

`theme.name === "Ubuntu.Components.Themes.SuruDark"`

So, can we resolve this conversation?

Reply:
Nice find :+1: